docs/reference-guide/query-modules/implement-custom-query-modules/api/python-api.md

```diff
@@ -468,6 +468,28 @@ Bases: `object`
 
 Annotate a resulting Record’s field as deprecated.
 
+## class mgp.Log()
+Bases: `object`
+
+Class for logging.
+
+### info(out:str)
+Logs a message `out` on `INFO` log level.
+
+### debug(out:str)
+Logs a message `out` on `DEBUG` log level.
+
+### error(out:str)
+Logs a message `out` on `ERROR` log level.
+
+### trace(out:str)
+Logs a message `out` on `TRACE` log level.
+
+### warning(out:str)
+Logs a message `out` on `WARNING` log level.
+
+### critical(out:str)
+Logs a message `out` on `CRITICAL` log level.
```

Comment:
It would be great if you could update this to look like a current version of docs: https://memgraph.com/docs/memgraph/reference-guide/query-modules/api/python-api#add_label 
If you look at this part of newer docs, you can notice that Python API has a different comments and look.  Here is the[ PR420](https://github.com/memgraph/memgraph/pull/420) at memgraph, where we made the changes, after that I generated new docs. This 2.4 version branch was created before that so it is the old way. The same goes for your code in mgp.py, please add the comments in format similar to PR420.

Reply:
Fixed